### DISCUSSION TITLE
Centralise Issues

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/ark#2980] [Q&A] [answered]
Hi everyone, hope you're doing well.

First, thank you for all the hard work on these projects (Chakra, Zag, Panda, Ark, and ParkUI).

I wanted to ask if there’s any plan to centralize GitHub issues for some of the projects in the future. Sometimes, it’s hard to know where to open an issue, whether it’s for Zag, Chakra, Ark, or Panda. Maybe having a single repository for reporting issues, with tags or labels to specify the project (like how the Flutter team does it), would be helpful.

**Comment by segunadebayo** [comment, marked as answer]
Feel free to open an issue in any of the projects, we’ll transfer them to the correct one accordingly. 

We don’t have plans to create another repository for issues.
